events/forms.py

```diff
@@ -17,15 +12,20 @@ class EventForm(forms.ModelForm):
     """
     class Meta:
         model = Event
-        fields = ('name', 'start_date', 'end_date', 'location',
-                  'description', 'highlights', 'accessibility',
-                  'ticket_info', 'additional_notes', 'website')
-        widgets = {"start_date": DateInput(),
-                   "end_date": DateInput()}
+        widgets = {
+            "start_date": DateInput(),
+            "end_date": DateInput(),
+        }
+        fields = (
+            'name', 'start_date', 'end_date', 'location',
+            'description', 'highlights', 'accessibility',
+            'ticket_info', 'additional_notes', 'website'
+        )
         labels = {
             'start_date': 'Start Date',
             'end_date': 'End Date',
+            'accessibility': 'Accessibility (Optional)',
             'ticket_info': 'Ticket Information',
-            'additional_notes': 'Additional Notes'
-        }
-
+            'additional_notes': 'Additional Notes (Optional)',
+            'website': 'Website'
+        }
```

Comment:
Can you add the blank line at the end of the file back in please? (PEP8 compliance and all that jazz)